app/src/main/java/com/nervousfish/nervousfish/service_locator/IServiceLocatorCreator.java

```diff
@@ -0,0 +1,9 @@
+package com.nervousfish.nervousfish.service_locator;
+
+
+/**
+ * Defines the interface of a class that can create a new {@link IServiceLocator}.
+ */
+public interface IServiceLocatorCreator {
```

Comment:
It's possible that we have multiple Service Locators like we had in SEM. The modules have a reference to a service locator, so we'll have to use an interface to provide support for multiple service locators.